eo/src/eoRanking.h

```diff
@@ -48,19 +48,7 @@ class eoRanking : public eoPerf2Worth<EOT> // false: do not cache fitness
     */
     eoRanking(double _p = 2.0, double _e = 1.0) : pressure(_p), exponent(_e)
     {
-        if (pressure <= 1.0)
-        {
-            std::string msg = "eoRanking: pressure must be > 1.0";
-            eo::log << eo::errors << "ERROR: " << msg << std::endl;
-            throw eoException(msg);
-        }
-
-        if (exponent > 2.0)
-        {
-            std::string msg = "eoRanking: exponent must be <= 2.0";
-            eo::log << eo::errors << "ERROR: " << msg << std::endl;
-            throw eoException(msg);
-        }
+        assert(1 < pressure and exponent <= 2);
```

Comment:
Doc says "pressure in (1,2]", but assert says something different. Which one is right?